playbooks/common_57_major_upgrade_from.yml

```diff
@@ -71,6 +73,9 @@
       - Percona-Server-client-57
     when: ansible_os_family == "RedHat"
 
+  - name: check telemetry (enabled)
+    command: /package-testing/check_telemetry.sh ps -e  
```

Comment:
Add disabled telemetry check:
1) Add `PERCONA_TELEMETRY_DISABLE:1` variable on nstall Percona Server 5.7 lines 61 and 75
2) Change command to `/package-testing/check_telemetry.sh ps -d`